src/composables/usePool.ts

```diff
@@ -287,11 +310,14 @@ export function usePool(pool: Ref<AnyPool> | Ref<undefined>) {
   const isComposableStablePool = computed(
     (): boolean => !!pool.value && isComposableStable(pool.value.poolType)
   );
+  const isDeepPool = computed(
+    (): boolean => !!pool.value && isDeep(pool.value)
+  );
   const isStableLikePool = computed(
     (): boolean => !!pool.value && isStableLike(pool.value.poolType)
   );
   const isComposableStableLikePool = computed(
-    (): boolean => !!pool.value && isComposableStableLike(pool.value.poolType)
+    (): boolean => !!pool.value && isDeep(pool.value)
```

Comment:
This is incorrect, it should remain as it was:
```suggestion
    (): boolean => !!pool.value && isComposableStableLike(pool.value.poolType)
```